tasks/gitlab_helpers.py

```diff
@@ -281,3 +283,57 @@ def print_entry_points(ctx):
     print(len(entry_points), 'entry points:')
     for entry_point, config in entry_points.items():
         print(f'- {color_message(entry_point, Color.BOLD)} ({len(config)} components)')
+
+
+def compute_gitlab_ci_config_diff(ctx, before: str, after: str):
+    """
+    Computes the full configs and the diff between two git references.
+    The base commit where after is compared to is the LCA commit between before and after.
+    """
+
+    before_name = before or "merge base"
+    after_name = after or "local files"
+
+    # The before commit is the LCA commit between before and after
+    before = before or DEFAULT_BRANCH
+    before = ctx.run(f'git merge-base {before} {after or "HEAD"}', hide=True).stdout.strip()
```

Comment:
We can use [this](https://github.com/DataDog/datadog-agent/blob/main/tasks/libs/common/git.py#L57) I guess